crates/forge/tests/integration/roll.rs

```diff
@@ -158,7 +158,6 @@ fn roll_complex() {
                 let roll_checker1 = IRollCheckerDispatcher { contract_address: contract.deploy(@ArrayTrait::new()).unwrap() };
                 let roll_checker2 = IRollCheckerDispatcher { contract_address: contract.deploy(@ArrayTrait::new()).unwrap() };
 
-                let old_block_number1 = roll_checker1.get_block_number();
```

Comment:
Same as above. Maybe we should update the test rather than removing the value 🤔

Reply:
The elect test was copied from this one; so it's just an unused variable, the test does test what it's supposed to 👍